### PR TITLE
AP_Scripting: fix some more of the easy luacheck warnings

### PR DESCRIPTION
**Pull request by IamPete1**
Slowly getting more of our scripts to pass.

### CODE REVIEW

File: libraries/AP_Scripting/examples/OOP_example.lua [hunk 1]
```diff
@@ -25,7 +24,6 @@ local function PIFF(kFF,kP,kI,iMax)
    local _kFF = kFF
    local _kP = kP or 0.0
    local _kI = kI or 0.0
-   local _kD = kD or 0.0
```
Comment (IamPete1) [inline libraries/AP_Scripting/examples/OOP_example.lua:28]: libraries/AP_Scripting/examples/OOP_example.lua:27:10: unused variable _kD

File: libraries/AP_Scripting/examples/plane-doublets.lua [hunk 2]
```diff
@@ -65,7 +61,7 @@ function doublet()
             pre_doublet_mode = vehicle:get_mode()
             -- are we doing a doublet on elevator or rudder? set the other controls to trim
             local doublet_choice_pwm = rc:get_pwm(DOUBLET_CHOICE_CHANNEL)
-            local trim_funcs = {}
+            local trim_funcs
```
Comment (IamPete1) [inline libraries/AP_Scripting/examples/plane-doublets.lua:64]: libraries/AP_Scripting/examples/plane-doublets.lua:64:19: value assigned to variable trim_funcs is unused

File: libraries/AP_Scripting/examples/plane-doublets.lua [hunk 3]
```diff
@@ -19,8 +18,6 @@ local DOUBLET_TIME = 500 -- period of doublet signal in ms
 -- flight mode numbers for plane https://mavlink.io/en/messages/ardupilotmega.html
 local MODE_MANUAL = 0
 local MODE_FBWA = 5
-local MODE_FBWB = 6
-local MODE_RTL = 11
```
Comment (IamPete1) [inline libraries/AP_Scripting/examples/plane-doublets.lua:23]: libraries/AP_Scripting/examples/plane-doublets.lua:21:7: unused variable MODE_FBWB
    libraries/AP_Scripting/examples/plane-doublets.lua:22:7: unused variable MODE_RTL

File: libraries/AP_Scripting/examples/plane-doublets.lua [hunk 4]
```diff
@@ -30,7 +27,6 @@ local K_RUDDER = 21
 local start_time = -1
 local end_time = -1
 local now = -1
-local desired_mode = -1
```
Comment (IamPete1) [inline libraries/AP_Scripting/examples/plane-doublets.lua:33]: libraries/AP_Scripting/examples/plane-doublets.lua:32:7: variable desired_mode is never accessed

File: libraries/AP_Scripting/examples/test_load.lua [hunk 5]
```diff
@@ -23,9 +22,9 @@ function test_load()
       return
    end
    -- run the code within a protected call to catch any errors
-   local success, err = pcall(f)
+   local success, err2 = pcall(f)
```
Comment (IamPete1) [inline libraries/AP_Scripting/examples/test_load.lua:25]: libraries/AP_Scripting/examples/test_load.lua:25:19: variable err was previously defined on line 19

File: libraries/AP_Scripting/applets/runcam_on_arm.lua [hunk 6]
```diff
@@ -57,8 +55,6 @@ function update()
          rc:run_aux_function(RC_OPTION.RunCamControl, AuxSwitchPos.LOW)
       end
       delay = DELAY_LONG
-   else
-      delay = DELAY_SHORT
```
Comment (IamPete1) [inline libraries/AP_Scripting/applets/runcam_on_arm.lua:61]: libraries/AP_Scripting/applets/runcam_on_arm.lua:47:10: value assigned to variable delay is unused

**Review by peterbarker — APPROVED** [review]

**Pull request merged**